packages/@lightningjs/ui-components/src/components/MetadataCardContent/MetadataCardContent.js

```diff
@@ -64,7 +68,7 @@ export default class MetadataCardContent extends MetadataBase {
   }
 
   static get properties() {
-    return ['description', 'details', 'provider', 'title'];
+    return ['description', 'descriptionDetails', 'details', 'provider', 'title'];
```

Comment:
Can you run `yarn lint:fix`? This line is not formatted correctly, and there may be others that need to be adjusted, as well.